ProxyApp/Classes/IYesterdayRate.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace ProxyApp
+{
+    interface IYesterdayRate
```

Comment:
Rename the interface with 'IRateService'

Reply:
Named according to task conditions